rustdoc: allow list syntax for #[doc(alias)] attributes (#82846)
rust-lang/rust: src/doc/rustdoc/src/advanced-features.md

"big", search will show the `BigX` struct first.

There are some limitations on the doc alias names though: you can't use `"` or whitespace.

> You can add multiple aliases at the same time by using a list:

Reviewer: "by using a list" seems potentially confusing because people will probably assume you mean `= ["one", "two"]`.

Author: I checked and there is no attribute using the `[]` syntax in the std. In addition, we give an example literally the line after, so I think there is very little possibility for a misunderstanding here.

Reviewer: Good point :)

"big", search will show the `BigX` struct first.

There are some limitations on the doc alias names though: you can't use `"` or whitespace.

> Please note that you can also use the `doc(alias)` attribute using lists if you want
> multiple aliases on one item:

Reviewer 2: ```suggestion
You can add multiple aliases at the same time by using a list:
```
